packages/core/lib/testing/testrunner.js

```diff
@@ -77,11 +77,7 @@ TestRunner.prototype.initialize = function(callback) {
         function(err, data) {
           if (err) return callback(err);
 
-          var contracts = data
-            .map(JSON.parse)
-            .map(json =>
-              contract(json, self.config.networks[self.config.network].type)
-            );
+          var contracts = data.map(JSON.parse).map(json => contract(json));
```

Comment:
need to `provision` here.

Reply:
I don't think so... all this is used for is to get the ABI, which doesn't need the provider

Reply:
Ah, so I think the change here is from a cherry-picked commit from the `alphaTez` branch.

This is resolved, but there's still an instance or two that needs to be updated. Lemme update the main review comment.